youths/models.py

```diff
@@ -34,6 +34,7 @@ class YouthProfile(models.Model):
     profile = models.OneToOneField(
         Profile, related_name="youth_profile", on_delete=models.CASCADE
     )
+    membership_number = models.CharField(max_length=16, blank=True)
```

Comment:
The spec doesn't say anything about it but I'm not sure if wan't allow this to be Blank 🤔

Reply:
I adjusted the `post_save` signal handler a bit. It will now also generate a membership number when the value is empty. Allowing this to be blank is sort of required, if we would like to be able to create a youth profile though the Django admin, which afaik is the only place currently where the `blank` attribute is considered.